### PR TITLE
Add workshop slides and article

### PR DESCRIPTION
**Pull request by hauten**
This partially fulfills #147 with:

- directory for PDFs
- talk titles on workshop page linked to PDFs
- Comp news article links on home, news, and workshop pages
- workshop page updated with "coming soon" text for videos and contest submissions

### CODE REVIEW

**Comment by tzanio** [comment]
Should we remove https://github.com/mfem/web/pull/136/commits/06b3a025f39a2219b770bc67975fe89489f28710?

File: src/index.md [hunk 1]
```diff
@@ -86,10 +86,10 @@ a short description, and visualization software used.
 
 Date         | Message
 ------------ | -----------------------------------------------------------------
-Jul 29, 2021  | Version 4.3 [released](https://github.com/mfem/mfem/blob/v4.3/CHANGELOG).
+Nov 12, 2021 | [Article](https://computing.llnl.gov/about/newsroom/mfem-team-hosts-first-community-workshop) about our workshop.
```
Comment (tzanio) [inline src/index.md:89]: Awesome article 🎉 🎉 🎉

**Comment by hauten** [comment]
> Should we remove [06b3a02](https://github.com/mfem/web/commit/06b3a025f39a2219b770bc67975fe89489f28710)?

I figured @pazner could do it 😄 . A larger question is, should we rewrite the page in the past tense ("we held a workshop..."), remove the registration links, etc.

**Comment by tzanio** [comment]
> A larger question is, should we rewrite the page in the past tense ("we held a workshop..."), remove the registration links, etc.

For me, it is not critical to change it. I think we probably will have a separate page for all MFEM-related talks (from the workshops and the seminar) so it may be good to preserve this one as it was so we can reuse it next year :). We probably will rename it to `workshop21.md` or something when the time comes.

**Review by tzanio — APPROVED** [review]
Thanks @hauten !

**Review by pazner — APPROVED** [review]
Looks great! I added the vis contest entries and changed the paragraph about it on the workshop page in #150.

**Pull request merged**